egs/babel/s5c/local/syllab/generate_syllable_lang.sh

```diff
@@ -118,8 +118,8 @@ ln -s lex.syllabs2phones.disambig.fst $out/L_disambig.fst
 echo "Validating the output lang dir"
 utils/validate_lang.pl $out || exit 1
 
-sed -i'' 's/#1$//g' $lout/lexicon.txt
-sed -i'' 's/#1$//g' $lout/lexiconp.txt
+perl -i'' -pe 's/#1$//g' $lout/lexicon.txt
+perl -i'' -pe 's/#1$//g' $lout/lexiconp.txt
```

Comment:
Two notes here:
1. Use just `-i`, not `-i''`.
2. The same program can be applied to multiple files with `-i`, so this can be done to avoid copy/paste
```
perl -i -pe 's/#1$//g' $lout/lexicon.txt $lout/lexiconp.txt
```